README.md

```diff
@@ -30,6 +30,54 @@ Any of the extensions listed below can be mixed into a class by `include`-ing `H
 
 ### Coercion
 
+Three systems are available for coercion: ActiveModel types, dry-types, and hashie's builtin types. The use of a specified coercion system is determined by exactly which Coercion module you include in your Hashie class. Including the default `Hashie::Extensions::Coercion` module will use Hashie's builtin type system.
```

Comment:
I think links to the other two libraries' source would be good here too. `[ActiveModel](https://github.com/rails/rails/tree/master/activemodel)` and `[dry-types](https://github.com/dry-rb/dry-types)`.

Also, uppercase Hashie please.